.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: go
+
+go:
+  - 1.9.2
+
+env:
+  global:
+   - ARCH=$(go env GOARCH)
```

Comment:
we should also have "go env GOOS" , otherwise "docker push $REPO:$TAG" will not work as we are making the docker image using "ARCH:=${XC_OS}_${XC_ARCH}"/.